packages/tupaia-web/src/features/ExportSettings/ExportDescriptionInput.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { TextField } from '@material-ui/core';
+import { useExportSettings } from './ExportSettingsContext';
+import { ExportSettingLabel } from './ExportSettingLabel';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const ExportDescriptionInputArea = styled(TextField)`
+  & .MuiOutlinedInput-root {
+    border-radius: 4px;
+  }
+  & .MuiOutlinedInput-notchedOutline {
+    border: 0.1rem solid ${({ theme }) => theme.palette.text.secondary};
+  }
+  & .MuiOutlinedInput-root.Mui-error .MuiOutlinedInput-notchedOutline {
+    border: 0.1rem solid #f76853;
+  }
+  & .MuiInputBase-root {
+    padding: 0.5em;
+    background: none;
+    color: white;
+  }
+  margin-block-start: 1em;
+`;
+
+const ExportDescriptionFooter = styled.div<{
+  charLimitReached: boolean;
+}>`
+  display: flex;
+  align-self: end;
+  justify-content: space-between;
+  margin-top: 5px;
+  color: ${({ charLimitReached }) => (charLimitReached ? '#f76853' : '#ffffff')};
```

Comment:
```suggestion
  color: ${({ charLimitReached, theme }) => (charLimitReached ? '#f76853' : theme.palette.text.primary)};
```

Can we also put the error colour in a constant somewhere since we use it in two places?